internal/pkg/service/common/etcdop/iterator/iterator.go

```diff
@@ -23,12 +23,16 @@ type Definition struct {
 }
 
 type Iterator struct {
-	config          config
 	ctx             context.Context
 	opts            []op.Option
+	client          etcd.KV
+	sort            etcd.SortOrder
+	pageSize        int
+	revision        int64
+	fromSameRev     bool
 	err             error
-	start           string         // page start prefix
-	end             string         // page start prefix
+	start           string         // page start key
+	end             string         // page end key
 	page            int            // page number, start from 1
```

Comment:
I removed the config field,  because it wasn't clear which fields are actually used here.